stylelint.config.js

```diff
@@ -1,12 +1,281 @@
 module.exports = {
```

Comment:
Wir könnten die Configs für so ein file auch noch auslagern, was denkst du dazu? 
dann könnten wir das bei allen zukünftigen Projekten wiederverwenden

Reply:
@tschuems Ja, das war auch mein Vorschlag für die Boilerplate. Dazu gibt es schon einen Task.

Reply:
@tschuems Wenn die Config sonst passt, kann ich ein separates Modul vorbereiten, dass wir hier und in der Boilerplate verwenden können.

Reply:
@slavanga daran erinnere ich mich :) 
Da schon als Task definiert, kann man diesen auch mergen 👍